handlers/api_eth1.go

```diff
@@ -973,6 +1009,10 @@ func getValidatorExecutionPerformance(queryIndices []uint64) ([]types.ExecutionP
 			producerReward = mevBribe
 		}
 
+		if block.Time.AsTime().After(firstEpochTime) {
+			result.PerformanceTotal = result.PerformanceTotal.Add(result.PerformanceTotal, producerReward)
+			result.Performance365d = result.Performance365d.Add(result.Performance365d, producerReward)
+		}
```

Comment:
Do we want to add the data from the current day to the results?
The BIDS makes it sound like they should be ignored.

Reply:
You are right, we also need to consider Equal